aws/terraform/infra/variables.tf

```diff
@@ -0,0 +1,27 @@
+variable "app" {
+  type        = string
+  default     = "metaflow-infra"
+  description = "Name of the application"
+}
+
+variable "env" {
+  type        = string
+  default     = "dev"
+  description = "The environment for this stack to be created in. Used for the tfstate bucket and naming scope of resources."
+}
+
+variable "aws_region" {
+  type        = string
+  description = "AWS region we will deploy to."
+}
+
+variable "base_s3_data_sets_bucket_name" {
+  type        = string
+  description = "Base name of S3 bucket to create to store external datasets/artifacts."
+}
+
+variable "vpc_flow_log_s3_destination" {
+  type        = string
+  description = "The ARN of the S3 bucket to receive VPC flow logs."
```

Comment:
Is this a requirement?

Reply:
it is a requirement for our vpcs. Here it has a default value of an empty string which results in no flow log destination being set.

Reply:
it just occurred to me that with the modularization, we definitely don't need this. I think we should yank it.

Reply:
Happy to yank both the variable and where its used. 

Just to make sure I understand, are you saying that you'll be able to reinstate this functionality on your end using say a private module?

Reply:
yeah.. my perspective is that the infra/metaflow/sagemaker-notebook projects are example infrastructure that we wouldn't just copy directly into our project, so we don't need to mirror the CNN specific items like flow logs in the vpc